glusterd2/commands/peers/deletepeer.go

```diff
@@ -38,15 +39,15 @@ func deletePeerHandler(w http.ResponseWriter, r *http.Request) {
 	// Check whether the member exists
 	p, err := peer.GetPeerF(id)
 	if err != nil {
-		logger.WithError(err).Error("failed to get peer")
+		if err == errors.ErrPeerNotFound {
```

Comment:
What's wrong with returning the `err` back to client directly ? I'm sure that has the same or similar error string. I prefer not to handle `ErrPeerNotFound` in a special way